python/lsst/ts/xml/data/sal_interfaces/MTHexapod/MTHexapod_Commands.xml

```diff
@@ -147,6 +147,95 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTHexapod</Subsystem>
+    <EFDB_Topic>MTHexapod_command_moveInSteps</EFDB_Topic>
+    <Description>Move the Hexapod to a specified position and orientation in steps. This sets the uncompensated position; if compensation is enabled, the actual position will be slightly different.</Description>
+    <item>
+      <EFDB_Name>x</EFDB_Name>
+      <Description>x position</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>um</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>y</EFDB_Name>
+      <Description>y position</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>um</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>z</EFDB_Name>
+      <Description>z position</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>um</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>u</EFDB_Name>
+      <Description>Rotation about x</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>v</EFDB_Name>
+      <Description>Rotation about y</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>w</EFDB_Name>
+      <Description>Rotation about z</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>stepSizeXY</EFDB_Name>
+      <Description>Step size of x-, y-direction. Put 0 if you want to do the movement in a single step.</Description>
```

Comment:
Should this be step size of x and y direction?

Reply:
This is used for the movement of individual x-direction or y-direction, or both. Thanks!